Makefile

```diff
@@ -77,13 +77,17 @@ test: static unit
 
 unit:
 	cargo build --all-targets
-	RUST_BACKTRACE=1 RUST_LOG=error cargo test -- --nocapture
+	RUST_BACKTRACE=1 RUST_LOG=info cargo test -- --nocapture
+	#--TODO--Once libdatachannel is the default, we'll want to keep
+	#        go-pion tested for a while until we're ready to deprecate it
+	#RUST_BACKTRACE=1 RUST_LOG=info cargo test --no-default-features --features backend-go-pion --manifest-path crates/tx5-connection/Cargo.toml -- --nocapture
```

Comment:
Can you not run tests for the entire workspace?

Reply:
The line that remains is testing the entire workspace with the default feature.
The tests I commented out represent the 2 crates that are actually affected by having a different backend feature, so there is no need to run the other crates again.